### ISSUE TITLE
Coloring Relevant Information

### ISSUE THREAD

**Issue by wtheisen** [wtheisen/commandChan#55]
Right now all of the text in the frames, other than greentext and quotes, is just grey. It might be nice to, on 4chan for instance, color the same post number a consistent color throughout a thread. Just coloring relevant information so the text becomes easier to parse would be nice.

**Comment by 287mdsahil** [comment]
Would like to work for this

**Comment by lukelima** [comment]
Is this issue still relevant ? I would like to work on this

**Comment by wtheisen** [comment]
Definitely, I think it’d really go a long way towards making things look much nicer!
